src/blueprint/from_config/operators/call.rs

```diff
@@ -0,0 +1,70 @@
+use crate::blueprint::*;
+use crate::config;
+use crate::config::{Config, Field, GraphQLOperationType};
+use crate::directive::DirectiveCodec;
+use crate::try_fold::TryFold;
+use crate::valid::Valid;
+
+pub fn update_call(
+  operation_type: &GraphQLOperationType,
+) -> TryFold<'_, (&Config, &Field, &config::Type, &str), FieldDefinition, String> {
+  TryFold::<(&Config, &Field, &config::Type, &str), FieldDefinition, String>::new(
+    move |(config, field, type_of, name), b_field| {
+      let Some(call) = &field.call else {
+        return Valid::succeed(b_field);
+      };
+
+      if validate_field_has_resolver(name, field, &config.types).is_succeed() {
+        return Valid::fail(format!(
+          "@call directive is not allowed on field {} because it already has a resolver",
+          name
+        ));
+      }
+
+      Valid::from_option(call.query.clone(), "call must have query".to_string())
```

Comment:
Query or Mutation both are supported.

Reply:
It was supported first but I ended up deleting it for lack of use case. 

Can you provide a .graphql that would use it?

I've tried to add `@call` in a mutation inside the Mutation type, but that field was not handled by this workflow

Reply:
It would look exactly like the one for `query` except the call operator will use `mutation` instead of query.

Reply:
I can have a "nested" mutation?

Something like: insertUser and insertPost, but on User type I add a posts property that will hit the /posts when I trigger insertUser. Does that make sense?

Reply:
You can, why not. It's just a function call and should be allowed. An example could be you have a list of users to create, but you call createUser mutation one by one using call and expr. I think this way we are getting closer to building a turning complete DSL :) so let's not stop!

Reply:
but does the current implementation already support it? if not, I agree that it's a valid use case, but it's not part of the issue this PR aims to solve

Reply:
I've tried to add a use case/test here (without `@call` operator) but does I did not had success doing so.

I first got the message that `Post.user` should be an `input` (rather than a `type`), then after changing it to `UserInput` I got the message that I can't have an input on an output value. I did not tried to add `@http` operator on the `PostInput.user` type, and now I don't have the `.graphql` here anymore, but if you have a valid `.graphql` file that uses nested mutation (with only `@http` operator), I'll do my best to support it on the new `@call` operator.

In case we don't have it, we could create a new issue to add support (potentially to both the plain resolver – `@http`, `@grpc`, etc – and the `@call` operator that we'll introduce on this PR.